src/univariate/discrete/binomial.jl

```diff
@@ -73,6 +73,22 @@ function mode(d::Binomial{T}) where T<:Real
 end
 modes(d::Binomial) = Int[mode(d)]
 
+function median(rv::Binomial)
+    bound = min(rv.p, 1-rv.p)
+    rv_mean = mean(rv)
+    floor_mean = floor(Int, rv_mean)
```

Comment:
In principle one could also use
```suggestion
    fractional_mean, integer_mean = modf(rv_mean)
```
to get the fractional and the integer part of the mean.
But I am unsure whether there would be any benefit (performance-wise or implementation-wise).

Reply:
I made some changes that are equivalent in spirit, I think.